explorer/src/Config.elm

```diff
@@ -24,6 +24,7 @@ type alias Config =
     , isHoveringFileUpload : Bool
     , selectedFiles : List File
     , infoLabel : InfoLabel
+    , sliderInputValue : String
```

Comment:
could this be float? Would that simplify/remove the need of the number parsing below?

Reply:
We did it like this because the Html input type requires a string, but we could also parse it the other way I guess:) Any opinions on what way of parsing is preferred?

Reply:
My preference is float to string, because that is safer, less error handling and I feel like it actually is a number value that we use, so it feels cleaner to me to have a number and when you display it then you convert it to a string.
Vs having a string that you hope is a number

Reply:
Agreed :D we'll change it to Float